src/data/prod/lex.lean

```diff
@@ -105,4 +106,7 @@ instance linear_order (α β : Type*) [linear_order α] [linear_order β] : line
   decidable_eq := lex.decidable_eq _ _,
   .. prod.lex.partial_order α β }
 
+instance is_well_order' (α β : Type*) [has_lt α] [has_lt β]
```

Comment:
Why the prime?

Reply:
There's already a `prod.lex.is_well_order`, which states `is_well_order r α → is_well_order s β → is_well_order (prod.lex s r) (β × α)`. These are def-eq, as seen from the proof here.